imsim/batoid_wcs.py

```diff
@@ -103,6 +104,8 @@ def __init__(
         self.rh = self.H2O_pressure/es  # relative humidity
         self.wl = self.wavelength * 1e-3  # nm -> micron
 
+        self.logger = logger
```

Comment:
I don't see where this is actually used, but I'd recommend not doing this.  Loggers aren't pickleable, so this immediately makes this class not pickleable either.  Maybe doesn't matter, but if we can avoid doing that, it's probably better.
Generally I prefer to pass in the logger to any function where we would want to use it.  Then if we call if from multiple processes, each process would be able to log to its own logger, not some global self.logger.

Reply:
Ah, okay.  I was using this (possibly incorrectly/poorly) while debugging this issue and thought it might be nice to keep around for the future.  But if it messes with the pickleability then happy to get rid of it.